libraries/client/client_linter_callback.py

```diff
@@ -152,6 +152,8 @@ def deploy_if_conversion_finished(s3_results_key, identifier):
                 build_log['multiple'] = True
 
             ClientLinterCallback.upload_build_log(build_log, "final_build_log.json", output_dir, s3_results_key)
+            if not multiple_project:
```

Comment:
FYI @PhotoNomad0 : This was added because the build log for single projects was NOT getting the linter warnings, thus wasn't having a status of "warning". It is only for multi-part in the door43_deploy lambda function that copies final_build_log.json to build_log.json, triggering the whole deploy again, yet didn't do that for a single job, thus it never actually got the final_build_log.json. I couldn't figure out the best way to do this though, as doing it this way also triggers the deploy process twice for a single build. Thoughts?

Reply:
Oddly they were getting their warning messages, but status wasn't being updated to "warnings" except in the final_build_log.json.